### PR TITLE
edit(RJMR-254): jsonschema without patternProperties

### PR DESCRIPTION
**Pull request by FilippoRezzonico**
[RJMR-254](https://makeitapp.atlassian.net/browse/RJMR-254)

### CODE REVIEW

File: reflect.go [hunk 1]
```diff
@@ -477,18 +475,8 @@ func (r *Reflector) reflectMap(definitions Definitions, t reflect.Type, st *Sche
 		st.Description = r.lookupComment(t, "")
 	}
 
-	switch t.Key().Kind() {
-	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:
```
Comment (fredmaggiowski) [inline reflect.go]: Why did you remove these lines?

Comment (FilippoRezzonico) [reply reflect.go]: I have added them back
